composer.json

```diff
@@ -66,9 +66,13 @@
 		"1_phpstan-typo3_fork": {
 			"type": "vcs",
 			"url": "https://github.com/linawolf/phpstan-typo3.git"
+		},
+		"2_typo3-typoscript-lint_fork": {
+			"type": "vcs",
+			"url": "https://github.com/linawolf/typo3-typoscript-lint.git"
 		}
 	},
-	"prefer-stable": true,
+	"minimum-stability": "dev",
```

Comment:
I would stick to "prefer-stable: true" and add additionally "minimum-stability" for now. When I am not mistaken this avoids v11.5 installed from branch and only installs dev branches when no released version with the given constraint is available.